Samples/kmp-app/shared/src/androidMain/kotlin/sample/kpm_app/Proxy.kt

```diff
@@ -0,0 +1,23 @@
+package sample.kpm_app
+
+import android.content.Context
+import io.sentry.kotlin.multiplatform.ContextProvider
+import io.sentry.kotlin.multiplatform.Sentry
+
+class Proxy {
+    /**
+     * For manual initialization
+     */
+    fun init(dsn: String, resolveContext: ()-> Context) {
+        ContextProvider.init(resolveContext)
+        Sentry.init(dsn)
```

Comment:
If this is Android, it should be `SentryAndroid.init(...)`

Reply:
In the [KMP SDK Android](https://github.com/buenaflor/sentry-kotlin-multiplatform/blob/main/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryAndroid.kt) it calls the `SentryAndroid.init(...)`, the KMP SDK will automatically route to the correct implementation, so in the future on iOS if I call `Sentry.init(...)` it would route to the iOS portion of the KMP SDK which would be `SentrySDK.start(...)` I believe

Reply:
Ah gotcha, makes sense, all good.